modules/frontend/queue/queue.go

```diff
@@ -140,19 +140,17 @@ FindQueue:
 		}
 
 		// Pick next request from the queue.
-		for {
```

Comment:
The `for` in line 135 could also be removed, and the block here encapsulated with `if queue != nil{}`. That'll get rid of the linter warning.